### ISSUE TITLE
Help; Required To Port SEGI to HD SRP

### ISSUE THREAD

**Issue by deltamish** [sonicether/SEGI#24]
Hey, 
With the HD SRP around the corner and having checked it out myself. Unity is moving closer to photorealism. and closer to Unreal like graphics . 

And our current project is making heavy use of SEGI . I opneded up the SEGI and shader script. It seemed pretty straight forward at first. But I am having trouble to see where you are acessing Unity Standard shader "_MainTex,_Albedo" and what not 

I was wondering if you could provide us on an insight on how I could go around port in to HD SRP   
Just some API example wher I can modify the texture being sampled and maybe the render Passes 

Thanks a ton :)

**Comment by noise2014** [comment]
plz up up !

**Comment by DuckyDuckling** [comment]
>Unreal like graphics

**Comment by JasonWeise** [comment]
Yes please, this would be a great addition to our work in HD SRP also.

**Comment by smokelore** [comment]
Bump! It's been awhile and I'm really excited to use HD SRP, but don't want to leave SEGI behind!

**Comment by thecrazy** [comment]
I think we are going to have to port it ourselves

**Comment by badjester** [comment]
Did anyone tried to port it? Any success?

**Comment by NasosTsoukalas** [comment]
A fully real time voxel based system for URP based on SEGI is now available in the asset store, [LUMINA GI: Voxel Based Global Illumination.](https://assetstore.unity.com/packages/tools/particles-effects/lumina-gi-real-time-voxel-global-illumination-226660) 
It is for URP and a HDRP version is also almost ready.

The system is made to work directly with [Sky Master ULTIMATE ](http://u3d.as/agN)URP Ethereal Volumetric Lighting system, for a complete atmospheric scattering and lighting simulation.

**Comment by FormalSnake** [comment]
dat price tho

**Comment by NasosTsoukalas** [comment]
> 

Hi, can contact in my Discord for current offers on this - https://discord.gg/X6fX6J5
